apps/prairielearn/src/ee/pages/institutionAdminCourse/institutionAdminCourse.ts

```diff
@@ -39,4 +43,52 @@ router.get(
   }),
 );
 
+const UpdateEnrollmentLimitsBodySchema = z.object({
+  __action: z.literal('update_enrollment_limits'),
+  yearly_enrollment_limit: z.union([z.literal(''), z.coerce.number().int()]),
+  course_instance_enrollment_limit: z.union([z.literal(''), z.coerce.number().int()]),
+});
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    const course = await queryRow(
+      sql.select_course,
+      {
+        institution_id: req.params.institution_id,
+        course_id: req.params.course_id,
+      },
+      CourseSchema,
+    );
```

Comment:
Not asking for a change here, but I really like the explicit authz flow that we are now using in PT, where this would be something like
```
const user = authnUser(res.params);
const institution = authzInstitutionAsAdmin(req.params.institution_id, user);
const course = authzCourseInInstitution(req.params.course_id, institution);
```

Reply:
Yeah, I like that too! Won't change just yet; I think we should switch to that pattern all at once within the `ee` portion of the project for consistency.

Reply:
Doing this in `ee` first is a great plan.